robottelo/ui/locators/base.py

```diff
@@ -1779,6 +1779,38 @@
                   "/span[contains(@class, 'info-value')]"),
     "package.content_file": (By.XPATH, "//div[contains(., '%s')]"),
 
+    # Puppet Module
+    "puppet.module_name": (By.XPATH, "//a[contains(., '%s')]"),
+    "puppet.author": (
+        By.XPATH, "//span[text()='Author']/../"
```

Comment:
I bet you tried, but... Isn't there a way not to stick to text values (which will depend on locale settings, of course)?